Week-4/Ex3.js

```diff
@@ -0,0 +1,19 @@
+let obj = {
```

Comment:
Give proper names and build that as a habit

Reply:
Noted. Thanks.